content/docs/guides/deploy/analytics_amp/analytics-vendors.md

```diff
@@ -0,0 +1,232 @@
+---
+$title: "Analytics Vendors"
+$order: 4
+toc: true
+---
+[TOC]
```

Comment:
Oh, shiny, we have a TOC now? Seeing the vendors in one place will be incredibly helpful